osf_tests/management_commands/test_move_egap_regs_to_provider.py

```diff
@@ -23,7 +23,7 @@ class TestEGAPMoveToProvider:
 
     @pytest.fixture()
     def egap_provider(self):
-        return RegistrationProviderFactory(name=settings.EGAP_PROVIDER_NAME)
+        return RegistrationProviderFactory(name=settings.EGAP_PROVIDER_NAME, _id='egap')
```

Comment:
In order for this to work you are going to have to add a the lines:
```python
        if _id:
            obj._id = _id
```
to `osf_tests/factories.py` on line 356.

This results from an oversight when the registration provider factory had to be modified to prevent legacy tests from breaking. Looking at the blame, I think I did that because I though the _id was being set as a guid somehow.